gprofiler/main.py

```diff
@@ -506,7 +504,7 @@ def parse_cmd_args() -> Any:
         dest="profile_api_version",
         default=None,
         choices=["v1"],
```

Comment:
Add new versions?

Reply:
See my comment in https://github.com/Granulate/gprofiler/pull/299#issuecomment-1063493241, no need for a new version here.